test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/io/OutputStreamWriterTest.scala

```diff
@@ -27,20 +26,23 @@ class OutputStreamWriterTest {
     bos.write(1)
     osw.write("ABC")
     assertFalse(bos.flushed)
+    assertEquals(1, bos.size)
```

Comment:
I do not understand this: This basically asserts that `osw` is not allowed to propagate the string until we call flush, no? That's definitely not true...

Reply:
Ah, good point. You're right, that's too restrictive.